test/units/config/test_manager.py

```diff
@@ -142,3 +143,16 @@ def decrypt(self, value, filename=None, obj=None):
 
         actual_value = ensure_type(vault_var, value_type)
         assert actual_value == "vault text"
+
+
+@pytest.mark.parametrize(("key", "expected_value"), (
+    ("COLOR_UNREACHABLE", "bright red"),
+    ("COLOR_VERBOSE", "rgb013"),
+    ("COLOR_DEBUG", "gray10")))
```

Comment:
If this is desired to be merged as a regression test before fixing, maybe use the xfail marker?
```suggestion
@pytest.mark.parametrize(
    ("key", "expected_value"),
    (
        ("COLOR_UNREACHABLE", "bright red"),
        pytest.param(
            "COLOR_VERBOSE", "rgb013",
            marks=pytest.mark.xfail(reason="https://github.com/ansible/ansible/issues/78607", strict=True),
        ),
        pytest.param(
            "COLOR_DEBUG", "gray10",
            marks=pytest.mark.xfail(reason="https://github.com/ansible/ansible/issues/78607", strict=True),
        ),
    ),
)
```